### DISCUSSION TITLE
为什么我这里 questions/index.html 页面没有生成侧边栏

### DISCUSSION THREAD

**Discussion by tjx666** [web-infra-dev/rspress#755] [General]
代码：https://github.com/tjx666/front-end-interview/tree/main/docs/questions

效果：

<img width="1440" alt="image" src="https://github.com/web-infra-dev/rspress/assets/41773861/de21391a-6e12-4976-9a69-b1d1f9e9c37b">

**Comment by sanyuan0704** [comment]
![image](https://github.com/web-infra-dev/rspress/assets/39261479/a0363e04-1a73-44c5-8dfc-f4d1b574023c)_meta.json 加一下 index
